<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>246</width>
    <height>155</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>246</width>
    <height>155</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>246</width>
    <height>155</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>模式选择</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../../../Pictures/icon.png</normaloff>../../../../Pictures/icon.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="easyButton">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>30</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>简单模式</string>
    </property>
   </widget>
   <widget class="QPushButton" name="proButton">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>80</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>专业模式</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
